frontends/ol-utilities/package.json

```diff
@@ -16,6 +16,7 @@
     "@faker-js/faker": "^9.0.0",
     "api": "workspace:*",
     "classnames": "^2.3.2",
+    "currency-symbol-map": "^5.1.0",
```

Comment:
I added this as a convenience for getting the correct symbol for a currency code, but if preferable I could just copy the mapping of codes to symbols to our own code base instead.